site/en/api_docs/cc/_book.yaml

```diff
@@ -0,0 +1,19 @@
+upper_tabs:
+# Tabs left of dropdown menu
+- include: /_upper_tabs_left.yaml
+
+# Dropdown menu
+- name: API
+  path: /api_docs/
```

Comment:
If we're keeping versioned cc and java docs, we either need to add versions/{version}/ at some point, or rely on relative paths.